README.md

```diff
@@ -49,6 +49,27 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   roslaunch godel_surface_detection kinect2.launch
   ```
 
+### Applications network mode(Only simulation)
+
+- Run blending demo in full simulation mode (simulated robot and sensor)
+  
+  Server: 
+  If it's the first RobotCell instance,run following program: 
+  ```
+  roslaunch godel_irb2400_support irb2400_blending_server.launch namespace:=[robot namespace] first_robot:="true"
+  ```
+  If it's not the first robotCell instance, run following program:
+  ```
+  roslaunch godel_irb2400_support irb2400_blending_server.launch namespace:=[robot namespace]
+  ```
+  Client:
+  Cd into the 'src' directory of your catkin workspace and run the following:
+  ```
+  cd godel
+  . irb2400_client.sh [ROS Master IPaddress] [robot namespace]
```

Comment:
While using a shell script works well, these arguments can be passed to a launch file.  Within the launch file you can use the [substitution arguments](http://wiki.ros.org/roslaunch/XML) feature in order unfold the argument's values.